proto/lib/config/deviceconfiguration.proto

```diff
@@ -24,4 +24,5 @@ message DeviceConfiguration {
     repeated ObservedFolder ignored_folders            = 14;
     repeated ObservedFolder pending_folders            = 15;
     int32                   max_request_kib            = 16 [(ext.goname) = "MaxRequestKiB", (ext.xml) = "maxRequestKiB", (ext.json) = "maxRequestKiB"];
+    string                  remote_gui_port            = 17 [(ext.goname) = "RemoteGUIPort", (ext.xml) = "remoteGUIPort", (ext.json) = "remoteGUIPort", (ext.default) = "0"];
```

Comment:
Why is this a string?

Reply:
I imagine that I would eventually have to cast it into a string in order to concatenate it to an address, but there is no critical reason beyond preference.